packages/ibm-products-styles/src/components/FilterPanel/_filter-panel-checkbox-with-overflow.scss

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright IBM Corp. 2024, 2024
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+/* stylelint-disable function-no-unknown */
+/* stylelint-disable max-nesting-depth */
+
+// Standard imports.
+@use '../../global/styles/project-settings' as c4p-settings;
+@use '../../global/styles/mixins';
+
+@use '@carbon/layout/scss/convert' as *;
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/type';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+$block-class: #{c4p-settings.$pkg-prefix}--filter-panel-checkbox-with-overflow;
+$carbon: #{c4p-settings.$carbon-prefix};
+
+// BUG FIX: hide small (unrelated?) rectangle that appears when showing
+// the OverflowMenu, but doesn't seem to belong or serve a function.
+html
```

Comment:
Even though this is a work around, I don't think we should be setting styles on all overflow menus. This will set styles on overflow menus anyone has on a page, even those outside of the `FilterPanel`.  Can we scope this to at least overflow menus inside of the `$block-class`?

Reply:
My bad. My career-long workaround jumped in without thinking :)